src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Protocol/DelegatedTypes.cs

```diff
@@ -10,7 +10,8 @@
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Protocol;
 
 internal record DelegatedDiagnosticParams(
-    VersionedTextDocumentIdentifier HostDocument);
+    VersionedTextDocumentIdentifier HostDocument,
+    System.Guid CorrelationId);
```

Comment:
Add a using directive for `System` and then change to:

```suggestion
    Guid CorrelationId);
```

Reply:
adding Using System would make Range conflict with System.Range.